packages/react/src/use-render/useRender.ts

```diff
@@ -7,11 +7,13 @@ const emptyObject = {};
 
 /**
  * Returns an object with a `renderElement` function that renders a Base UI element.
+ *
+ * @public
```

Comment:
I am a bit confused, why do we need both @public and @private tags on hooks. Was the @internal on hooks a left over? For example `packages/react/src/utils/useAnimationsFinished.ts` has the @internal tag.

Reply:
Yeah, this was unnecessary. Hooks are considered internal by default. I removed the redundant tags.